### ISSUE TITLE
Fix generator to import from snake_case model name

### ISSUE THREAD

**Issue by oleoneto** [oleoneto/django-clite#20]
When running `D g model car-brand`, the model `CarBrand` is created in the module `car_brand.py`.

The import statement in `models/__init__.py` should be:
```python
from .car_brand import CarBrand
```

Instead, the current implementation imports:
```python
from .card-brand import CarBrand
```

The latter syntax is invalid.

**Comment by oleoneto** [comment]
This happens in `admins`, `serializers`, and `viewsets`.

**Comment by oleoneto** [comment]
Fixed in 601fc871c55879b87eb0630938c868a0523adbcc
